conda_build/noarch_python.py

```diff
@@ -55,8 +65,11 @@ def handle_file(f, d):
         os.unlink(path)
 
     # The presence of .so indicated this is not a noarch package
-    elif f.endswith('.so'):
-        _error_exit("Error: Shared object file found: %s" % f)
+    elif f.endswith(('.so', '.dll', '.pyd', '.exe', '.dylib')):
+        if f.endswith('.exe') and f.startswith('Scripts'):
```

Comment:
Yeah ... this is about what I did at first, but the `xx-script.py` could already have been processed and moved. Maybe keep track of these then?